Help CMake find ruby library if it is build statically

Signed-off-by: Jonas Karlsson <jonas@gobolinux.org>
--- qt4-qtruby-1.4.9/cmake/modules/FindRUBY.cmake.old	2008-03-24 21:06:48.000000000 +0100
+++ qt4-qtruby-1.4.9/cmake/modules/FindRUBY.cmake	2008-03-24 20:43:08.000000000 +0100
@@ -60,7 +60,7 @@
   	)
 
 	FIND_LIBRARY(RUBY_LIBRARY
-		NAMES ruby${RUBY_VERSION} ruby
+		NAMES ruby${RUBY_VERSION} ruby ruby-static
 		PATHS ${RUBY_POSSIBLE_LIB_PATH} ${RUBY_RUBY_LIB_PATH}
   	)
 
